android/src/main/java/com/instabug/reactlibrary/RNInstabugNetworkLoggerModule.java

```diff
@@ -0,0 +1,84 @@
+package com.instabug.reactlibrary;
+
+
+import static com.instabug.apm.configuration.cp.APMFeature.APM_NETWORK_PLUGIN_INSTALLED;
+import static com.instabug.apm.configuration.cp.APMFeature.CP_NATIVE_INTERCEPTION_ENABLED;
+import static com.instabug.apm.configuration.cp.APMFeature.NETWORK_INTERCEPTION_ENABLED;
+
+import androidx.annotation.NonNull;
+
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactMethod;
+import com.instabug.apm.InternalAPM;
+import com.instabug.reactlibrary.utils.EventEmitterModule;
+import com.instabug.reactlibrary.utils.MainThreadHandler;
+
+
+public class RNInstabugNetworkLoggerModule extends EventEmitterModule {
```

Comment:
seems like this module is not emitting events to RN do we still need to extend it from `EventEmitterModule`?

Reply:
@kholood-ea for future use